java/DEVELOPER.md

```diff
@@ -0,0 +1,255 @@
+# Developer Guide
+
+This document describes how to set up your development environment to build and test the GLIDE for Redis Java wrapper.
+
+### Development Overview
+
+The GLIDE for Redis Java wrapper consists of both Java and Rust code. Rust bindings for the Java Native Interface are implemented using [jni-rs](https://github.com/jni-rs/jni-rs), and the Java JAR is built using [Gradle](https://github.com/gradle/gradle). The Java and Rust components communicate using the [protobuf](https://github.com/protocolbuffers/protobuf) protocol.
+
+### Build from source
+
+#### Prerequisites
+
+Software Dependencies
+
+-   git
+-   GCC
+-   pkg-config
+-   protoc (protobuf compiler) >= 26.1
+-   openssl
+-   openssl-dev
+-   rustup
+-   Java 11
+
+**Install protobuf compiler (necessary for all systems)**
```

Comment:
All or all except mac? This conflicts with line 64

Reply:
Actually instruction hardcodes `linux`. Make more flexible instructions or add a note that there is an example.